Content.Server/Mobs/StandingStateHelper.cs

```diff
@@ -61,13 +61,13 @@ public static bool Standing(IEntity entity)
             return true;
         }
 
-        public static void DropAllItemsInHands(IEntity entity)
+        public static void DropAllItemsInHands(IEntity entity, bool doMobChecks = false)
```

Comment:
Why does this default to `false`? It seems a bit inconsistent with the rest of the methods.

Reply:
I could change it to `true` for consistency, but when I first wrote the method, my intention was to have a way to drop all items in the hands of a mob when the mob fell down (so the mob check wouldn't really make sense to be `true` by default)
I'll change it though.